yaksh/templates/yaksh/message.html

```diff
@@ -0,0 +1,51 @@
+<div class="msg_history">
+{% for message in room_messages %}
+<div class="incoming_msg">
+	<div class="incoming_msg_img"> <img src="{{ URL_ROOT }}/static/yaksh/images/user-profile.png"> </div>
+	<div class="received_msg">
+		<div class="received_withd_msg">
+		  <p>{{message.message}}</p>
+		  <small>{{message.sender}}</small>
+		  <span class="time_date">{{message.timestamp}}</span>
+		</div>
+	</div>
+</div>
+<br>
+{% endfor %}
+</div>
+<div class="type_msg">
+	<div class="input_msg_write">
+		<form method="POST" id="post-form">
+			<input id="post-text" type="text" class="write_msg" placeholder="Type a message" required />
+			<button class="msg_send_btn" type="button"><i class="fa fa-paper-plane-o" aria-hidden="true"></i></button>
+		</form>
+	</div>
+</div>
+<script type="text/javascript">
+	$(document).ready(function(e){
+		function post_message(){
+			console.log($('#post-text').val());
+			$.ajax({
+				url:"{% url 'yaksh:send_message' %}",
+				method:'POST',
+				data:{
+					room_id: room_id,
+					course_id: course_id,
```

Comment:
No need to send room_id and course_id in post data. Send it with `{% url 'yaksh:send_message' %}`